build/BuildContext.cs

```diff
@@ -19,7 +19,7 @@ public BuildContext(ICakeContext context) : base(context)
     {
         var repositoryUrl = context.Argument("build-repository", DefaultRepositoryUrl);
         var buildConfiguration = context.Argument("build-configuration", "Release");
-        BuildOutput = context.Argument("build-output", "artifacts");
+        BuildOutput = context.Argument("build-output", "Artifacts");
```

Comment:
Let the artifacts folder be called with a starting small letter was intentional on my part, I wanted to slowly resort the repo into:
- artifacts
- build
- src
- tests

etc. instead of our current setup.

Reply:
I had to change this because it was inconsistent across the repo. I'll have to update all the references to use lowercase if that is the intention, Not sure how this worked with mixed case on Linux in the first place.

Reply:
Oh yea, I forgot we use the artifacts folder for the build output as well.

I guess I'll try to fix it up tomorrow in a separate PR.